README.md

```diff
@@ -28,11 +28,14 @@ The site was built for the CS98 course at Dartmouth College.
 
 ## Run Locally
 1. Start up a local server: ```python manage.py server```
-2. Start up the scraper: ```python manage.py scrape```
+2. In another terminal session,
```

Comment:
Do you think we should explain the noqueue thing and this? How do you think we should do that?